### ISSUE TITLE
Dof not blending with transparent background

### ISSUE THREAD

**Issue by giagitom** [molstar/molstar#1239]
Here is a comparison with and without background

![image](https://github.com/user-attachments/assets/944abdfd-4abf-49c1-9977-8be479cde01e)

![image](https://github.com/user-attachments/assets/34f72d62-8843-420a-8af5-2561d73e1475)

I also notice artifacts with snapshots
![image](https://github.com/user-attachments/assets/552eeab9-131d-452d-87b5-6a271130e21a)
![image](https://github.com/user-attachments/assets/99aa88d0-a119-4fb6-8e4d-65bee7b463f0)

**Comment by arose** [comment]
@giagitom still an issue?

**Comment by giagitom** [comment]
Yes, same as before

**Comment by giagitom** [comment]
Still an issue, blurr isn't applied on transparent areas.

**Transparent bg**
<img width="1511" height="1122" alt="Image" src="https://github.com/user-attachments/assets/2fc18705-d6f0-47e7-8c03-650c270abbb3" />

**Opaque bg**
<img width="1511" height="1122" alt="Image" src="https://github.com/user-attachments/assets/ccf7e8c8-99ab-4e35-aff7-d1103e516e92" />


When on transparent bg, I also notice some darkening effect on places where blurring should be more intense. (SSAO is off here)

<img width="1087" height="1098" alt="Image" src="https://github.com/user-attachments/assets/50cd0352-3471-4def-b153-4f04066cbda1" />

**Comment by arose** [comment]
@giagitom with your recent changes, is this still an issue?

**Comment by giagitom** [comment]
Yeah, still an issue, no more artifacts on transparent objects, but the main issue remains no dof blending with transparent background and darkening of regions out of focus.   

<img width="1124" height="890" alt="Image" src="https://github.com/user-attachments/assets/2cdd6bc0-5d4d-4d30-bddd-deb8032dd1f6" />
